common.mk

```diff
@@ -355,7 +355,7 @@ buildroot: optee-os optee-rust
 		$(DEFCONFIG_FTPM) \
 		--br-defconfig out-br/extra.conf \
 		--make-cmd $(MAKE))
-	@$(MAKE) $(br-make-flags) -C ../out-br all
+	@export PATH=$(HOME)/.cargo/bin:$(PATH) && $(MAKE) $(br-make-flags) -C ../out-br all
```

Comment:
Even though `export` seems to work I wonder if `env` wouldn't be preferable.

Reply:
`env` applies the environment to the specified command only, while `export` modifies the environment that the shell passes to any subsequent sub-process. That's more or less the same here because the shell will exit as soon as the `$(MAKE) ...` command exits (make spawns a new shell for each line in a recipe).

Reply:
We should rather do `source "$HOME/.cargo/env"` here to setup path for Rust toolchain. It worked for me since I have that added as part of my `~/.bashrc`.

Reply:
> We should rather do `source "$HOME/.cargo/env"` here to setup path for Rust toolchain. It worked for me since I have that added as part of my `~/.bashrc`.

Good point.